interface/forms/prior_auth/templates/prior_auth/general_new.html

```diff
@@ -1,64 +1,93 @@
+
+<script>
+	$(function () {   
+	
+		$('.datetimepicker').datetimepicker({
+			{/literal}
+			{php} $datetimepicker_timepicker = false; {/php}
+			{php} $datetimepicker_showhours = false; {/php}
+			{php} $datetimepicker_formatInput = true; {/php}
+			{php} require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); {/php}
+			{literal}
+		});
+	
+	});
+</script>
 {/literal}
 </head>
-<body bgcolor="{$STYLE.BGCOLOR2}">
-<p><span class="title">Prior Authorization Form</span></p>
-<form name="prior_auth" method="post" action="{$FORM_ACTION}/interface/forms/prior_auth/save.php">
-<input type="hidden" name="csrf_token_form" value="{$CSRF_TOKEN_FORM|attr}">
-<table>
-<tr>
-	<td>Prior Authorization Number</td><td><input type="text" size="35" name="prior_auth_number" value="{$prior_auth->get_prior_auth_number()|attr}"></td>
-</tr>
-<tr>
-	<td><br /><br />Comments</td>
-</tr>
-<tr>
-	<td colspan="2"><textarea name="comments" value="{$prior_auth->get_comments()|attr}" wrap="virtual" cols="75" rows="8">{$prior_auth->get_comments()|text}</textarea></td>
-</tr>
-{if $VIEW != true}
-<tr>
-	<td><br /><br /><input type="submit" name="Submit" value="Save Form">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="{$DONT_SAVE_LINK}" class="link">[Don't Save]</a></td>
-</tr>
-{/if}
-</table>
-<input type="hidden" name="id" value="{$prior_auth->get_id()|attr}" />
-<input type="hidden" name="activity" value="{$prior_auth->get_activity()|attr}">
-<input type="hidden" name="pid" value="{$prior_auth->get_pid()|attr}">
-<input type="hidden" name="process" value="true">
+<body>
+	<div class="container mt-3">
+        <div class="row">
+            <div class="col-12">
+				<h2>{xlt t="Prior Authorization"}</h2>
+                <form name="prior_auth" class="form-horizontal" method="post" action="{$FORM_ACTION}/interface/forms/prior_auth/save.php" onsubmit="return top.restoreSession()">
+				<input type="hidden" name="csrf_token_form" value="{$CSRF_TOKEN_FORM|attr}">
+				<fieldset>
+					<legend>{xlt t="Details"}</legend>
+					<div class="form-group mt-3">
+						<div class="row">
+    					    <div class="col-sm-4">
+							    <label for="prior_auth_number" class="col-form-label">{xlt t="Number"}:</label>
+							    <input type="text" class="form-control" size="35" name="prior_auth_number" value="{$prior_auth->get_prior_auth_number()|attr}">
+						    </div>	
+						    <div class="col-sm-4">
+							    <label for="date_from" class="col-form-label">{xlt t="From"}:</label>
+							    <input type='text' size='10' class='form-control datetimepicker' name='date_from' id='date_from' value='{$prior_auth->get_date_from()|date_format:"%Y-%m-%d"|attr}' />
+						    </div>
+						    <div class="col-sm-4">	
+							   <label for="date_to" class="col-form-label">{xlt t="To"}:</label>
+							   <input type='text' size='10' class='form-control datetimepicker' name='date_to' id='date_to' value='{$prior_auth->get_date_to()|date_format:"%Y-%m-%d"|attr}' />
```

Comment:
rec removing the `date_format:"%Y-%m-%d"`in above 2 lines since not necessary and just make it more complicate to support internationalization in future